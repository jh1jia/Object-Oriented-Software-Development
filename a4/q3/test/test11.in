stdin dropfirst 3 dropfirst 3 allcaps allcaps count e
dropfirst 989 333 count 2 doublewords
sample.txt allcaps count E a.txt conut.txt
