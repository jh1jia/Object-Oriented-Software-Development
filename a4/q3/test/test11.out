RST
WORDS
.TXT
S
TXT
